packages/schemas/src/v3/connection.schema.ts

```diff
@@ -62,6 +62,7 @@ const schema = {
             "type": "string",
             "pattern": "^[\\w.-]+$"
           },
+          "default": [],
```

Comment:
nit: is there a reason we're doing this for only orgs and not the other fields?

Reply:
nevermind I see that this is the generated file and the schema adds them to all of them, so probably just an old version of the generated files or something